test/integration/PciManagerTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
```

Comment:
When running the integration test I receive the following error message:

`Use of undefined constant CONFIG_PATH - assumed 'CONFIG_PATH'`

Reply:
So is this back to normal in PHPUnit?